web/pom.xml

```diff
@@ -0,0 +1,76 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <name>MSKCC CMO MetaDB Web</name>
+  <description>MetaDB Web</description>
+  <artifactId>web</artifactId>
+  <version>0.1.0</version>
+  <packaging>jar</packaging>
+  <parent>
+    <groupId>org.mskcc.cmo.metadb</groupId>
+    <artifactId>master</artifactId>
+    <version>0.1.0</version>
+  </parent>
+
+  <dependencies>
+    <dependency>
+      <groupId>org.mskcc.cmo.metadb</groupId>
+      <artifactId>service</artifactId>
+      <version>${project.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>io.springfox</groupId>
+      <artifactId>springfox-swagger2</artifactId>
+      <version>2.7.0</version>
+      <scope>compile</scope>
+    </dependency>
+    <dependency>
+      <groupId>io.springfox</groupId>
+      <artifactId>springfox-swagger-ui</artifactId>
+      <version>2.7.0</version>
+      <scope>compile</scope>
+    </dependency>
+     <dependency>
+      <groupId>org.springframework.boot</groupId>
+      <artifactId>spring-boot-starter-web</artifactId>
+      <exclusions>
+        <exclusion>
+            <groupId>ch.qos.logback</groupId>
```

Comment:
change the indentation for these 2 lines to 2 spaces, I think they're at 4 spaces right now